packages/manager/modules/carbon-calculator/src/dashboard/carbon-balance-composition/template.html

```diff
@@ -4,11 +4,58 @@ <h4 data-translate="carbon_calculator_dashboard_carbon_balance_title"></h4>
         class="row-fluid"
         data-translate="carbon_calculator_dashboard_carbon_balance_explanation"
     ></p>
-    <img
-        class="border-bottom pb-3"
-        data-ng-src="{{:: $ctrl.IMG_CARBON_BALANCE }}"
-        alt
-        title="{{:: 'carbon_calculator_dashboard_carbon_balance_img_title' | translate}}"
-        data-ng-attr-alt="{{:: 'carbon_calculator_dashboard_carbon_balance_img_title' | translate}}"
-    />
+    <div class="row pb-3 col-xs-12 col-sm-8 text-center">
```

Comment:
bootstrap schema don't allow row within col grid system.
col depend on row which is a container.

please have a look:
https://getbootstrap.com/docs/4.0/layout/grid/

Reply:
I split row and col